server/src/adapters/postgres/postgres.js

```
@@ -499,6 +499,20 @@ module.exports = {
     return pool.query('SELECT NOW() as now')
       .then(function (res) {
         console.log('Connected to postgres on', res.rows[0].now)
+
+        // Initialize pool with NODEPG_INITIAL_MIN_CLIENTS
+        var promises = _.times(parseInt(process.env.NODEPG_INITIAL_MIN_CLIENTS || '10'), () => {
+          return pool
+            .connect()
+            .then((connection) => {
+              return connection.release()
+            })
+            .catch((error) => {
+              console.log('Error in initializing pool: ', error)
```

Are we OK with silencing errors in the initial pool warm-up?

---

You're right, I've removed this `catch` block so that the error is handled in `app.js` instead.

---

Where is that change?

---

https://github.com/rayyanqcri/rayyan-store/pull/90/commits/fba4f9504b5cf9d8d3fa1a5631540cc3a0f2d3d9